predict_on_directory.py

```diff
@@ -0,0 +1,35 @@
+########################################################################################################################
+# Model
+########################################################################################################################
```

Comment:
inaccurate comment, delete, (or change and use triple quotes to provide a documentation on the command line usage)